tests/integration/conftest.py

```diff
@@ -359,6 +359,66 @@ def create_label():
         label.delete()
 
 
+@pytest.fixture
+def configured_batch_project_with_label(client, rand_gen, image_url,
+                                        batch_project, dataset, datarow,
+                                        wait_for_label_processing):
+    """Project with a batch having one datarow
+    Project contains an ontology with 1 bbox tool
+    Additionally includes a create_label method for any needed extra labels
+    One label is already created and yielded when using fixture
+    """
+    data_rows = [dr.uid for dr in list(dataset.data_rows())]
+    batch_project.create_batch("test-batch", data_rows)
+    editor = list(
```

Comment:
Looks like a lot of the code after this is shared between this and `configured_project_with_label`. Could you create a helper function for the ontology setup + label creation part?